docs/logging.md

```diff
@@ -0,0 +1,59 @@
+# Logging
+
+Using Bigflow you can easily add handlers to your loggers that will send all loggers calls to the Cloud Logging.
+In the CL you can filter logs per project id/logger name and workflow id. 
+
+## Getting started
+To use logging you have to install `bigflow` with `logger` module, `bigflow[logger]`.
+To create logger with the CL handler you have to use `configure_logging` method.
+
+```python
+from bigflow.logger import configure_logging
+project_id = 'some_project'
+logger_name ='some-logger'
+
+
+configure_logging(project_id, logger_name)
+```
+
+or if you prefer to filter in the CL those logs by workflow id
```

Comment:
this senstence doesn't sound good, maybe something like this: or, if you prefer to filter CL logs by workflow id:

Reply:
Done